ark/analysis/spatial_analysis_test.py

```diff
@@ -6,6 +6,32 @@
 from ark.analysis import spatial_analysis
 from ark.utils import synthetic_spatial_datagen
 
+import ark.settings as settings
+
+EXCLUDE_CHANNELS = [
+    "Background",
+    "HH3",
+    "summed_channel",
+]
+
+DEFAULT_COLUMNS = \
+    [settings.CELL_SIZE] \
+    + list(range(1, 24)) \
+    + [
+        settings.CELL_LABEL,
+        settings.AREA,
+        settings.ECCENTRICITY,
+        settings.MAJ_AXIS_LENGTH,
```

Comment:
I don't think these need to be global settings if their only purpose is to properly name the test data. We're never going to change the names that come directly from regionprops